BASE/.vsts/linux-build.yml

```diff
@@ -20,6 +22,11 @@ steps:
   displayName: install dotnet core 3.1
   inputs:
     version: 3.1.x
+
+- task: UseDotNet@2
```

Comment:
nit: i think we dont need to explictly install any .NET versions at all, as these linux machines come preinstalled.

Reply:
i added this because I was getting an error from this build that Net5.0 wasn't available.